### ISSUE TITLE
Sidebar focus integration test flaky

### ISSUE THREAD

**Issue by cathyxz** [ampproject/amphtml#16157]
This test flakes on Chrome Mobile Webview only. Possibly worth investigating. 

```
DESCRIBE => amp-sidebar
  DESCRIBE => sidebar focus
    DESCRIBE =>  
      IT => should focus on opener on close
        ✗ Error: Timeout waiting for wait for sidebar to open
            at poll (/tmp/testing/iframe.js:418:17 <- /tmp/d11da25ed565a736da041dcabf540743.browserify:100419:18)
●○○○○○○○○○○○○○○○●○○●
[20:48:00] Chrome Mobile WebView 44.0.2403 (Android 6.0.0): Executed 85 of 402 (Skipped 317) 1 FAILED
```

**Comment by ampprojectbot** [comment]
This issue hasn't been  updated in awhile. @cathyxz Do you have any updates?

**Comment by aghassemi** [comment]
test is still skipped

**Comment by aghassemi** [comment]
closing unskip x issues, we can find them in the code
